bootstrap/src/Main.hs

```diff
@@ -215,7 +215,8 @@ instance A.ToJSON SrcType where
 -------------------------------------------------------------------------------
 
 info :: String -> IO ()
-info msg = hPutStrLn stderr $ "INFO: " ++ msg
+info _msg = return ()
```

Comment:
is this on purpose?

Reply:
Yes... The bootstrap generation was uselessly verbose (dumping the whole json output and this list of all dependencies), part of what made finding this bug difficult because it kept getting hidden in the output.

And then I couldn't think of a better way to do this, and thought it might be fine for a tool such as this -- if someone is interested in the debug output they can easily change this.

Other options I've considered:
- leave this in, but redirect stderr from the Makefile -- hard to get right since stderr vs stdout is already tricky with using `cabal run` (we don't want to hide `cabal`'s stderr)
- remove the logging entirely
- disable the logging by default, add a `-v` flag to turn it on (seemed like overcomplicating things)

Happy for alternate suggestions (and also happy to just revert this part if preferred).

Reply:
i would go for 2 or 3, maybe using a env var would be slightly simple?
but current version would be fine too, it is a script at the end

Reply:
I'd be quite happy to remove the logging entirely -- will wait a while for further input then maybe make that change.

Reply:
I'd be fine with an extensive code comment that says `put hPutStrLn stderr $ "INFO: " ++ msg there to get detailed debug info`. Basically comment out the line and say why and when it's useful.

Reply:
sounds good, I'll make that change

Reply:
Turns out I'm not completely sure I understood your suggestion. I pushed a doc comment for the disabled `info` function, is this good?